paper.md

```diff
@@ -58,202 +58,195 @@ bibliography: paper.bib
-| | <p align="center">2</p> | <p align="center">Vectors, data frames, basic operations and use of functions</p>  | <p align="center">`tidyverse` [@tidyverse]</p> |
-| | <p align="center">3</p> | <p align="center">Exploratory analysis of datasets</p> | <p align="center">`tidyverse` |
-| | <p align="center">4</p> | <p align="center">Summary statistics and data visualization </p>| <p align="center">`tidyverse` |
-| | <p align="center">5</p> | <p align="center">Transforming datasets to different formats, reproducing existing figures</p> | <p align="center">`tidyverse`</p> |
-| <p align="center">Exploratory data analysis and statistics</p> | <p align="center">6</p> | <p align="center">Cleaning up and preprocessing of raw data</p> | <p align="center">`tidyverse`; `mice` [@mice]</p> |
-| | <p align="center">7</p> | <p align="center">Basic descriptive and inferential statistical techniques (e.g linear models)</p> | <p align="center">`tidyverse`; `car` [@car]; `psych` [@psych]; `multcomp` [@multcomp]</p> |
-| | <p align="center">8</p> | <p align="center">Linear mixed-effects models</p> | <p align="center">`tidyverse`; `plyr` [@plyr]; `lme4` [@lme4]; `lmerTest` [@lmerTest]</p> |
-| | <p align="center">9</p> | <p align="center">Randomization tests and data simulation</p>  | <p align="center">`tidyverse`; `reshape2` [@reshape2]; `EcoSimR` [@EcoSimR]</p> |
-| | <p align="center">10</p>| <p align="center">Multivariate statistics (e.g. PCA, MANOVA) </p>| <p align="center">`tidyverse`; `car`; `psych`; `multcomp`</p>|
-| | <p align="center">11</p>| <p align="center">Model selection and model averaging</p>  | <p align="center">`tidyverse`; `lme4`; `lmerTest`; `MuMIn` [@MuMIn]</p> |
-| <p align="center">Numerical models </p>| <p align="center">12</p>| <p align="center">Population modelling using differential equations</p>  | <p align="center">`tidyverse`; `deSolve` [@deSolve]</p> |
-| | <p align="center">13</p>| <p align="center">Time-series data and numerical models</p>  | <p align="center">`tidyverse`; `deSolve`</p> |
-| <p align="center">Collaboration and reproducibility</p> | <p align="center">14</p> | Scientific methods, working in groups and final project group formation</p>  | <p align="center">NA</p> |
-| | <p align="center">15</p>| <p align="center">Collaboration through Git and GitHub</p>  | <p align="center">NA</p> |
-| | <p align="center">16</p>| <p align="center">Git and GitHub, metadata and manuscript preparation in R Markdown</p>  | <p align="center">`knitr` [@knitr]; `rmarkdown` [@rmarkdown]</p> |
-
-Table: Overview of modules, lectures, and packages used throughout the course.
-
+The overarching objective of the course is to teach reproducible and
+collaborative quanititative research skills. The lessons are described in more
+detail in Table 1 and are designed into three overall submodules:
+
+1. Programming in R [@R], basic data wrangling, and visualizations (lessons 1-5).
```

Comment:
for consistency to line 122, s/visualizations/visualization

Reply:
Only the last element in a list should end in a period; non-final entries end in semi-colons.

Reply:
@joelostblom I agree as well. It is aesthetically more pleasing (my view of course) to have periods.